acceptance/tests/pxp_agent_version.rb

```diff
@@ -4,7 +4,8 @@
 
 agents.each do |agent|
   step 'cd into pxp-agent bin folder and check the version' do
-    on(agent, "cd #{pxp_agent_dir(agent)} && ./pxp-agent --version") do |result|
+    # Vendored Ruby needs to be on path. The "privatebindir" property of beaker host can be used for this
+    on(agent, "export PATH=\"#{agent['privatebindir']}:$PATH\" && #{pxp_agent_dir(agent)}/pxp-agent --version") do |result|
```

Comment:
The export is required so that our vendored components are on the path. Without it, pxp-agent will exit with errors about libs not being loadable